src/main/java/ch/psc/gui/SignUpController.java

```diff
@@ -51,9 +56,9 @@ private void initialize() {
                 buildControl();
             }
         });
-        flowControl.isDoneProperty().addListener((observable, old, newValue) -> finish());
+        flowControl.isDoneProperty().addListener((observable, old, newValue) -> SignUpController.this.finish());
         flowControl.isCanceledProperty().addListener((observable, old, newValue) -> {
-            if (newValue) cancel();
+            if (newValue) SignUpController.this.cancel();
         });
```

Comment:
SignUpController.this unnötig. Bitte rausnehmen

Reply:
weiss nicht woher das kommt. gelöscht